src/data/equiv/basic.lean

```diff
@@ -569,6 +569,16 @@ def subtype_subtype_equiv_subtype {α : Type u} (p : α → Prop) (q : subtype p
   λ⟨a, ha⟩, ⟨⟨a, ha.cases_on $ assume h _, h⟩, by { cases ha, exact ha_h }⟩,
   assume ⟨⟨a, ha⟩, h⟩, rfl, assume ⟨a, h₁, h₂⟩, rfl⟩
 
+/- A subtype of a sigma-type is a sigma-type over a subtype. -/
+def sigma_subtype {α : Type u} (p : α → Type v) (q : α → Prop) :
```

Comment:
The name should mention `equiv`. I suggest `subtype_sigma_equiv_sigma_subtype` or `subtype_sigma_equiv`

Reply:
I indeed messed up the order of `sigma` and `subtype`. I thought omitting `equiv` was fine in the `equiv` namespace, but the extra clarity can't hurt. I went for `subtype_sigma_equiv`.